[2024-02-13T06:52:57.712+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: deverlop_model.start scheduled__2024-02-13T06:52:44.272274+00:00 [queued]>
[2024-02-13T06:52:57.723+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: deverlop_model.start scheduled__2024-02-13T06:52:44.272274+00:00 [queued]>
[2024-02-13T06:52:57.724+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 1
[2024-02-13T06:52:57.737+0000] {taskinstance.py:2191} INFO - Executing <Task(_PythonDecoratedOperator): start> on 2024-02-13 06:52:44.272274+00:00
[2024-02-13T06:52:57.744+0000] {standard_task_runner.py:60} INFO - Started process 78 to run task
[2024-02-13T06:52:57.747+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'deverlop_model', 'start', 'scheduled__2024-02-13T06:52:44.272274+00:00', '--job-id', '4', '--raw', '--subdir', 'DAGS_FOLDER/task.py', '--cfg-path', '/tmp/tmpmcvh2mbt']
[2024-02-13T06:52:57.750+0000] {standard_task_runner.py:88} INFO - Job 4: Subtask start
[2024-02-13T06:52:57.800+0000] {task_command.py:423} INFO - Running <TaskInstance: deverlop_model.start scheduled__2024-02-13T06:52:44.272274+00:00 [running]> on host 3346c6dff994
[2024-02-13T06:52:57.875+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='rtn' AIRFLOW_CTX_DAG_ID='deverlop_model' AIRFLOW_CTX_TASK_ID='start' AIRFLOW_CTX_EXECUTION_DATE='2024-02-13T06:52:44.272274+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-02-13T06:52:44.272274+00:00'
[2024-02-13T06:52:57.878+0000] {logging_mixin.py:188} INFO - START JOBS
[2024-02-13T06:52:57.878+0000] {python.py:201} INFO - Done. Returned value was: None
[2024-02-13T06:52:57.886+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=deverlop_model, task_id=start, execution_date=20240213T065244, start_date=20240213T065257, end_date=20240213T065257
[2024-02-13T06:52:57.920+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-02-13T06:52:57.935+0000] {taskinstance.py:3280} INFO - 0 downstream tasks scheduled from follow-on schedule check
